website/blog/2023-03-24-databend-weekly-86.md

```diff
@@ -0,0 +1,150 @@
+  - name: dantengsky
+  - name: everpcpc
+  - name: leiysky
+  - name: lichuang
+  - name: mergify[bot]
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: xinlifoobar
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: yufan022
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### FlightSQL Handler in Progress
+
+Flight SQL is an innovative open database protocol that caters to modern architectures. It boasts a columnar-oriented design and provides seamless support for parallel processing of data partitions.
+
+The benefits of supporting FlightSQL include reducing serialization and deserialization during query execution, as well as easily supporting SDKs in different languages using predefined proto.
+
+Databend is actively working on supporting the FlightSQL Handler. If you're interested, visit the following materials.
+
+- [Issue #10745 | tracking: FlightSQL handler](https://github.com/datafuselabs/databend/issues/10745)
+- [PR #10732 | feat: basic impl for FlightSQL handler](https://github.com/datafuselabs/databend/pull/10732).
+
+### Natural Language to SQL
+
+Are you still struggling with how to write complex queries? By integrating with popular AI services, Databend provides you with an efficient built-in solution - the `AI_TO_SQL` function.
+
+With this function, instructions written in natural language can be converted into SQL query statements aligned with table schema. With just a few modifications (or possibly none at all), it can be put into production.
+
+```sql
+SELECT * FROM ai_to_sql(
+    'List the total amount spent by users from the USA who are older than 30 years, grouped by their names, along with the number of orders they made in 2022',
+    '<openai-api-key>');
+*************************** 1. row ***************************
+     database: openai
+generated_sql: SELECT name, SUM(price) AS total_spent, COUNT(order_id) AS total_orders
+               FROM users
+                        JOIN orders ON users.id = orders.user_id
+               WHERE country = 'USA' AND age > 30 AND order_date BETWEEN '2022-01-01' AND '2022-12-31'
+               GROUP BY name;
+```
+
+If you want to quickly experience our natural language to SQL function, you can try [deploying Databend](https://databend.rs/doc/deploy) or [registering for our cloud service](https://app.databend.com/). In addition, reading related materials can also help you learn more information:
```

Comment:
The function is now available on both Databend and Databend Cloud. To learn more about how it works, refer to the  following links: